### PR TITLE
add webserver bolt example

### PR DESCRIPTION
**Pull request by Joris29**
This adds a new example that provisions an EC2 instance with terraform and installs apache with Puppet Bolt

### CODE REVIEW

File: example-workflows/ec2-provision-and-configure-webserver/ec2-provision-and-configure-webserver.yml [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+      repository: https://github.com/Joris29/nebula-aws-provision.git
+
+- name: configure-apache-bolt
+  dependsOn:
+    - provision-ec2-with-terraform
+  image: projectnebula/bolt-run
+  spec:
+    type: plan
+    name: example::webserver
+    projectDir: example-workflows/ec2-provision-and-configure-webserver/Boltdir/
```
Comment (kyleterry) [inline example-workflows/ec2-provision-and-configure-webserver/ec2-provision-and-configure-webserver.yml]: We just added the ability to instruct bolt in the projectnebula/bold-run step image to download Puppet modules listed in a `Puppetfile` inside your `projectDir` directory. You can see some examples of that here: https://github.com/puppetlabs/nebula-infra-demo/blob/master/bolt/Puppetfile and here: https://github.com/puppetlabs/nebula-infra-demo/blob/master/workflow.yaml#L87.

The spec for `projectnebula/bolt-run` supports a new configuration called `installModules`:

```suggestion
    projectDir: example-workflows/ec2-provision-and-configure-webserver/Boltdir/
    installModules: true
```

**Comment by Joris29** [comment]
@kyleterry i get a notice or error: Could not find a Puppetfile at /workspace/repo/aws-workflow-example/example-workflows/ec2-provision-and-configure-webserver/Boltdir/Puppetfile

But it still installs the modules and configures apache somehow :)

**Comment by Joris29** [comment]
@kyleterry nvm i a was working with the wrong git repository

File: example-workflows/ec2-provision-and-configure-webserver/README.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
+<h1>Installs apache to an EC2 instance on AWS with bolt and provision with terraform</h1>
```
Comment (kyleterry) [inline example-workflows/ec2-provision-and-configure-webserver/README.md]: This README should be markdown, but raw HTML. Markdown would condense this considerably. Take a look at https://raw.githubusercontent.com/puppetlabs/nebula-workflow-examples/master/example-workflows/gke-provision-and-deploy-workflow/README.md

**Comment by Joris29** [comment]
@kyleterry readme is fixed

**Pull request merged**